src/server.js

```diff
@@ -208,6 +208,7 @@ function init(socket, client) {
 		socket.on(
 			"open",
 			function(data) {
+				client.lastActiveSocket = socket;
```

Comment:
This approach seems fine (at least temporarily) by me. I'll 👍 it unless other maintainers have other considerations.